### PR TITLE
ACJ Buff

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Buffs the ACJ

- ACJ now covers the head
- ACJ now grants complete immunity to fires
- ACJ makes you stupidly fast

DWI

### CODE REVIEW

File: code/modules/clothing/under/miscellaneous.dm [hunk 1]
```diff
@@ -140,12 +140,12 @@
 	desc = "it's a cybernetically enhanced jumpsuit used for administrative duties."
 	gas_transfer_coefficient = 0.01
 	permeability_coefficient = 0.01
-	body_parts_covered = UPPER_TORSO|LOWER_TORSO|LEGS|FEET|ARMS|HANDS
+	body_parts_covered = UPPER_TORSO|LOWER_TORSO|LEGS|FEET|ARMS|HANDS|HEAD
 	armor = list(melee = 100, bullet = 100, laser = 100,energy = 100, bomb = 100, bio = 100, rad = 100)
```
Comment (Iamgoofball) [inline code/modules/clothing/under/miscellaneous.dm]: >invincible armor

Balanced :tm:

**Comment by tigercat2000** [comment]
ACJ is an admin item right

**Comment by Fox-McCloud** [comment]
Yeah, the Administrative Cybernetic Jacket.

**Comment by SamCroswell** [comment]
:+1: 
ACJ is already OP as shit, and unused outside of the admin room.
Might as well seal the deal.

**Comment by Spacemanspark** [comment]
NO IS TOO OP AND WILL GET ABOOS.

...

:+1:

**Comment by Fox-McCloud** [comment]
Now with 100x more balance.

**Comment by SamCroswell** [comment]
Oh lord, slowdown -10 is sure to get annoying for the person wearing it.

**Comment by Spacemanspark** [comment]
Nuh uh.
SANIC SPEEDS.

**Pull request merged**